.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2.1
+jobs:
+  build_and_test:
+    parameters:
+      image:
+        type: string
+    docker:
+      - image: << parameters.image >>
+        environment:
+          MIX_ENV: test
+
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            mix local.hex --force
+            mix local.rebar --force
+            mix deps.get
+      - run: mix format --check-formatted
+      - run: mix compile --warnings-as-errors
+      - run: mix test
+      - run: mix credo
+
+      - run: mix coveralls
+      # TODO: `GITHUB_TOKEN` should be added to `bank_validator_br` CircleCI context before uncommenting the following step
```

Comment:
opa @adrianomitre , acredito que não vamos precisar do `mix coveralls.github` mais, já que essa configuração é pro sumário do coveralls aparecer no github action.

Acredito que se mudar a l26 pra `- run: mix coveralls.circle` E alterar o [mix.exs:L15](https://github.com/Xerpa/bank_validator_br/blob/master/mix.exs#L15) pra `"coveralls.circle": :test,` elemina a necessidade desse TODO.

Outra coisa. Você poderia criar o job do publish, pra quando for pra master? Só o esqueleto mesmo, só dando um `echo Deployed` pra depois eu colocar a parte do deploy em si :)

Reply:
Já adicionei no circle CI o token do coveralls.

Reply:
@itsmegrave Acho que podemos mergear esse PR e criar um outro com o "boilerplate" para o publish e deixar aberto até implementarmos o deploy de fato. O que acha?

Reply:
faz sentido @riosgabriel 👍🏾